### ISSUE TITLE
The name 'softmax loss function'  should be 'cross entropy loss function'

### ISSUE THREAD

**Issue by xiaochunrilihe** [ConstellationBJUT/Coursera-DL-Study-Notes#1]
https://github.com/ConstellationBJUT/Coursera-DL-Study-Notes/blob/947387c4a7cc99d62b708bd2edb4a2a8e391a9ca/class5_week1_sequence/lstm/lstm.py#L202

this should be cross entropy loss 交叉熵损失，而不是softmax损失，softmax只是一个映射函数

**Comment by bjjoy** [comment]
> https://github.com/ConstellationBJUT/Coursera-DL-Study-Notes/blob/947387c4a7cc99d62b708bd2edb4a2a8e391a9ca/class5_week1_sequence/lstm/lstm.py#L202
> 
> the right fomula is
> Loss = ln[ sum( pi^(yi) * log[ (1-pi) ^ (1- yi) ) ] = sum( yi * ln(pi) + (1- yi) * ln(1-pi) )

o? Softmax is used to multi classification, so the loss function is self.loss... Your loss function is two classification with LR. Is my understand right?
